src/parser/core/CASTABLE_WHILE_CASTING_SPELLS.ts

```diff
@@ -3,12 +3,12 @@ import SPELLS from 'common/SPELLS';
 export default [
   /**
    * This consists of spells that are able to be cast while casting. This is
-   * primarily used to ignore the below spells when determining whether a 
+   * primarily used to ignore the below spells when determining whether a
    * spell cast was cancelled since these do not interrupt casting.
    */
   SPELLS.FIRE_BLAST.id,
   SPELLS.SHIMMER_TALENT.id,
   SPELLS.ICE_FLOES_TALENT.id,
   SPELLS.COMBUSTION.id,
   SPELLS.SEARING_NIGHTMARE_TALENT.id,
-];
+] as number[];
```

Comment:
Rather than using `as`, you should do something like below, which types the export without force-casting it:

```typescript
const spells: number[] = [
 SPELLS.FIRE_BLAST.id,
 ...
];
export default spells;
```

Same for CASTS THAT ARENT CASTS

Reply:
Fixed - thanks!